### DISCUSSION TITLE
Is an internet connection required?

### DISCUSSION THREAD

**Discussion by VictorioBerra** [mcguirepr89/BirdNET-Pi#330] [Q&A]
I have a battery module for my pi. Could I take the pi on a camping trip and when I get back to wifi catch up on all the analysis?

**Comment by mcguirepr89** [comment]
An internet connection is _not_ required. 

[This guide is likely very helpful for you](https://github.com/mcguirepr89/BirdNET-Pi/wiki/How-to-use-BirdNET-Pi-in-the-field-without-home-router-access) as it will walk you through exactly how to run things offline anywhere you are. The only step that might be missing is setting the system time in "Tools" > "Settings" and then "Tools" > "Services" > "Restart All Services."

I'll have a pre-installed available eventually that will present a auto-hotspot to set things up offline anywhere with very few steps: 1) connect to "BirdNET-Pi" Hot Spot, 2) Set the Latitude and Longitude and System Time, 3) Restart All Services.

I'll let you know when that is available!

**Reply by VictorioBerra** [reply]
Thanks! Initial setup is not a concern I have wifi at home so I have plenty of time to prepare and configure.

**Reply by VictorioBerra** [reply]
@mcguirepr89 stupid question, are samples analyzed locally or are they sent to Cornell servers?

**Reply by mcguirepr89** [reply]
Not stupid!!! All analysis is done locally right on the Pi -- no info is sent out for analysis
